docs/getting-started/walk-through/advanced-hera-features.md

```diff
@@ -160,85 +162,104 @@ def echo_all(
 
 The fields allowed in the `Parameter` annotations are: `name`, `default`, `enum`, and `description`.
 
-##### Artifacts
+#### Artifacts
+
+> Note: Artifact annotations are only supported when used with the `RunnerScriptConstructor`.
 
-The improvement is even more noticeable for `Artifact`s. In Hera we are able to specify `Artifact`s in `inputs`:
+
+The feature is even more powerful for `Artifact`s. In Hera we are currently able to specify `Artifact`s in `inputs`, but
+have no way to programmatically link the given path to the code within the function:
```

Comment:
We have no way as in we Hera developers? Maybe "the given path is not programmatically linked to the code" would be better?